tasks/install_runner.yml

```diff
@@ -5,6 +5,7 @@
     state: directory
     mode: 0755
     owner: "{{ runner_user }}"
+    group: "{{ runner_user }}"
```

Comment:
Hi @domask , thanks for this PR. I agree with it. However, not all systems will have the group name the same as the user name. Therefore i would suggest to add a task into `collect_info.yml`  which would find the group id of `runner_user`. After that it could be used in `Create directory` and `Unarchive runner package` tasks.

Task which will find the user group (and user) IDs could look like this:
```
  - name: Find Runner User IDs
    ansible.builtin.getent:
      database: passwd
      key: "{{ runner_user }}"
```

and then replace owner and group values with this:

```    
owner: "{{ ansible_facts.getent_passwd[runner_user].1 }}"
group: "{{ ansible_facts.getent_passwd[runner_user].2 }}"
```
I guess this could work...